src/Config.php

```diff
@@ -909,6 +909,7 @@ protected function getDefaults()
                 'dbname'         => 'bolt',
                 'prefix'         => 'bolt_',
                 'charset'        => 'utf8',
```

Comment:
These defaults could change to utf8mb4 in future if you want to support multi byte characters out of box for non-technical users.